.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Bug report
+about: Something is not working as expected
+title: ''
+labels: 'bug, priority: medium'
+assignees: ''
+
+---
+
+### Describe the bug
+A clear and concise description of what the bug is.
```

Comment:
Maybe add these explanations/descriptions as comments?

Reply:
I did this on purpose. Now you have to replace them. We have comments in the GitLab templates and they usually kind of clutter the issue source and I looked into the larger projects to see what they are doing and seems to be more standard.